public/views/content/studentView.html

```diff
@@ -0,0 +1,3 @@
+<a ui-sref=".addEntry">+ Add new entry</a>
```

Comment:
Brenna, you can just branch off my branch and do stuff while this PR is
pending

On Sun, Apr 10, 2016 at 11:53 PM BrennaManning <notifications@github.com>
wrote:

> In public/views/content/studentView.html
> <https://github.com/davidabrahams/Spring-Initiative/pull/80#discussion_r59152039>
> :
>
> > @@ -0,0 +1,3 @@
> > +<a ui-sref=".addEntry">+ Add new entry</a>
>
> I suppose I just don't understand what the problem is, but it'd be nice to
> resolve it sooner since not merging is making it challenging to make much
> progress
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/davidabrahams/Spring-Initiative/pull/80/files/82c4e851bb880cbb3a20ef88419c47beea1ab840#r59152039>
>